addons/stock/views/res_config_settings_views.xml

```diff
@@ -62,7 +62,7 @@
                             </setting>
                         </block>
                         <block title="Shipping" name="shipping_setting_container">
-                            <setting id="stock_move_email" company_dependent="1" string="Email Confirmation" help="Send an automatic confirmation email when Delivery Orders are done">
+                            <setting id="stock_move_email" company_dependent="1" string="Email Confirmation" help="Send an automatic confirmation email when Delivery Orders or Dropships are done">
```

Comment:
Dropships only exists once `stock_dropshipping` is installed but... not sure we can help it :man_shrugging:

Reply:
Done. I have transferred code to the dropshipping module so it works properly.